components/gantry/client_test.go

```diff
@@ -142,6 +160,11 @@ func TestClient(t *testing.T) {
 		test.That(t, pos, test.ShouldResemble, pos2)
 		test.That(t, extra2, test.ShouldResemble, map[string]interface{}{"foo": "123", "bar": 234.})
 
+		homed, err := client2.Home(context.Background(), map[string]interface{}{"foo": 345, "bar": "456"})
+		test.That(t, err, test.ShouldNotBeNil)
```

Comment:
use the `ShouldContainSubstring` comparison or the `error.Is()` comparison, the `base` and `movementsensor` client and server files are what we eventually want these test files to look like.